blocks/large-promo-block/features/large-promo/default.jsx

```diff
@@ -99,12 +139,143 @@ const LargePromoItem = ({ customFields, arcSite }) => {
     }`,
 		}) || null;
 
-	return <LargePromoPresentation content={content} {...customFields} />;
+	const { editableContent, searchableField } = useEditableContent();
+	const { registerSuccessEvent } = useComponentContext();
+	const {
+		dateLocalization: { language, timeZone, dateTimeFormat } = {
+			language: "en",
+			timeZone: "GMT",
+		},
```

Comment:
I see you removed `dateTime` here.  If `dateLocalization` is ever undefined, it will still throw an error.

My comment here: https://github.com/WPMedia/arc-themes-blocks/pull/1411#discussion_r920164012

..should be resolved with:
```suggestion
		dateLocalization: { language, timeZone, dateTimeFormat } = {
			language: "en",
			timeZone: "GMT",
			dateTimeFormat: "LLLL d, yyyy 'at' K:m bbbb z",
		},
```

Reply:
@nschubach, thanks. Fixed it in [59045b2](https://github.com/WPMedia/arc-themes-blocks/pull/1411/commits/59045b235ff358c5095838ea6a37ad471a96752a)